tsconfig.json

```diff
@@ -37,5 +37,8 @@
       "shared/js/background/storage/https.es6.js",
       "shared/js/devtools/list-editor.es6.js",
       "shared/js/devtools/panel.es6.js"
-  ]
+  ],
+    "types": [
+        "chrome"
```

Comment:
This is fine for now but ideally we should type this interface, I'm guessing someone has already done this for DNR like we do for the webextensions-polyfil (which is kinda not helpful post MV3 or at least currently)

Reply:
Maybe it would be better to make it local to the dnr file for now as all MV2 code uses the browser namespace which is typed (with some legacy code exceptions).

Reply:
Yea, I'm using `@types/chrome` to type the interface. I tested and using the declarativeNetRequest APIs wrongly is caught. Defo agree about using the polyfill in the future though.